raiden/ui/cli.py

```diff
@@ -68,6 +68,13 @@
         type=str,
     ),
     click.option(
+        '--rpccorsdomain',
+        help='Comma separated list of domains to accept cross origin requests. \n'
+        '(localhost enabled by default)',
+        default=None,
```

Comment:
The problem is that localhost and 127.0.0.1 are recognised as different domains. Atleast that is what I observed during testing. But I can make this change.